src/Microsoft.ML.Transforms/Text/LineParser.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.ML.Transforms.Text
+{
+    public static class LineParser
+    {
+        private static readonly char[] _delimiters = { ' ', '\t' };
+
+#if NETSTANDARD2_0
```

Comment:
It seems to me it is worth beginning to multitarget an assembly if there is a significant win (like in this case). However I suggest partial classes rather than #if when dealing with entire methods, like here. That pattern has helped lower code complexity in corefx.

cc @eerhardt for thoughts

Reply:
That's fine, but I'd first like to understand *why* we think the divergence is necessary, since the divergence seems to increase our required test load. If there's a good reason, that's fine, but my suspicion is that it is mostly due to being unaware of the conveniences we ourselves had to build up to parse text files quickly. (E.g., `DoubleParser.TryParse`), which might all around lead to better performing code in both cases, not one.

Reply:
@danmosemsft @TomFinley I had no idea about the existence of `DoubleParser`. I have re-used it and now we have a simple implementation again.